scripts/installer.sh

```diff
@@ -321,6 +321,18 @@ main() {
 		exit 1
 	fi
 
+	TEST_URL="https://pkgs.tailscale.com/"
+	RC=0
+	TEST_OUT=$($CURL "$TEST_URL" 2>&1) || RC=$?
+	if [ $RC != 0 ]; then
+		echo "The installer needs to be able to reach Tailscale package server"
```

Comment:
I'd recommend simplifying the structure of this sentence, a number of our users in China rely on machine translation. pkgs.tailscale.com is blocked from China.

Suggest: "The installer cannot reach $TEST_URL, please make sure that your machine has internet access."

Reply:
Thanks, I've rephrased the message.